tools/ImportExamples.ps1

```diff
@@ -18,14 +18,45 @@ function Start-Generator {
         [string] $ManualExternalDocsUrl = "https://learn.microsoft.com/en-us/graph/api/user-get?view=graph-rest-1.0&tabs=powershell"
     )
 
-    $GraphMapping = @{
-        "v1.0" = "v1.0\examples"
-        "beta" = "beta\examples"
-    }
     if ($GenerationMode -eq "auto") {
-        $GraphMapping.Keys | ForEach-Object {
-            $graphProfile = $_
-            Get-FilesByProfile -GraphProfile $graphProfile -GraphProfilePath $GraphMapping[$graphProfile] -ModulesToGenerate $ModulesToGenerate 
+        $CommandMetadataContent = Get-Content $CommandMetadataPath | ConvertFrom-Json
+        $CommandMetadataContent | ForEach-Object {
+            $GraphCommand = $_.Command
+            $GraphModule = $_.Module
+            $UriPath = $_.Uri
+            $ExternalDocsUrl = $_.ApiReferenceLink
+            $ApiVersion = $_.ApiVersion
+            $ProfilePathMapping = "v1.0\examples"
+            if ($ApiVersion -eq "beta") {
+                $ProfilePathMapping = "beta\examples"
+            }
+            $ModulePath = Join-Path $PSScriptRoot "..\src\$GraphModule\$ProfilePathMapping"
+
+            $ExampleFile = "$ModulePath\$GraphCommand.md"
+            Test-Commands -Command $GraphCommand -CommandPath $ExampleFile
+            if ($null -ne $ExternalDocsUrl) { 
+                if (-not (Test-Path $ExampleFile)) {
+                    New-Item -Path $ExampleFile -ItemType File -Force
+                }
+                $IntuneUrl = $ExternalDocsUrl.Replace("intune-onboarding-", "")
+                $IntuneUrl = $MUrl.Replace("intune-mam-", "")
```

Comment:
Yes, they exist., however if you pick these two examples i.e. https://learn.microsoft.com/en-us/graph/api/intune-onboarding-user-get?view=graph-rest-1.0&tabs=http and https://learn.microsoft.com/en-us/graph/api/user-get?view=graph-rest-1.0&tabs=http, you'll notice that the intune prefixed covers less scenarios of the cmdlet usage compared to the other one.  Therefore, the script is giving preference to the non-intune prefixed one if it's reachable (returns a 200), otherwise it retains the original enpoint gotten from the external docs parameter in the open api file.

Reply:
I missed that we were falling back to the original endpoint if the "non-intune" docs URL doesn't exist. Thanks.